diff --git a/index.js b/index.js
index f72159e0ac7dcd95bb854776435ddc7112d8906a..6fb6b91609e8f2d2a4402018bd93715074fcb53a 100644
--- a/index.js
+++ b/index.js
@@ -1,12 +1,12 @@
 'use strict';
 
-var origSymbol = global.Symbol;
+// var origSymbol = global.Symbol;
 var hasSymbolSham = require('./shams');
 
 module.exports = function hasNativeSymbols() {
-	if (typeof origSymbol !== 'function') { return false; }
+//	if (typeof origSymbol !== 'function') { return false; }
 	if (typeof Symbol !== 'function') { return false; }
-	if (typeof origSymbol('foo') !== 'symbol') { return false; }
+//	if (typeof origSymbol('foo') !== 'symbol') { return false; }
 	if (typeof Symbol('bar') !== 'symbol') { return false; }
 
 	return hasSymbolSham();
